pcap_ex.c

```diff
@@ -27,15 +27,15 @@ int
 pcap_ex_immediate(pcap_t *pcap)
 {
 #ifdef _WIN32
-    return pcap_setmintocopy(pcap, 1);
+	return pcap_setmintocopy(pcap, 1);
```

Comment:
It seems like you have replaced spaces with `\t`. I see now you probably did this because the rest of the file has mixed `\t` and spaces.
I would say that given the fact that we use spaces everywhere else, we should probably also go for spaces in here too.